CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+**Fixed Bugs:**
+
+- Fix comparison of User provider home paths to ignore trailing slashes
```

Comment:
oh you shouldn't submit changelog entries, we generate those off of a process involving some autogeneration based on PR titles and some manual massaging of the result.  an you amend this commit to drop this and re-push?